reconcile/openshift_groups.py

```diff
@@ -112,41 +101,48 @@ def fetch_current_state(thread_pool_size, internal, use_jump_host):
     return oc_map, current_state, ocm_clusters
 
 
-def fetch_desired_state(oc_map, enforced_user_keys=None):
+def fetch_desired_state(
+    oc_map: ClusterMap, enforced_user_keys: Optional[list[str]] = None
+) -> list[dict[str, str]]:
     gqlapi = gql.get_api()
-    roles = expiration.filter(gqlapi.query(ROLES_QUERY)["roles"])
```

Comment:
You removed `expiration.filter`, was that on purpose?

Reply:
Oh that is a bug! `expiration.filter` is important. I didn't notice locally. Fixed now.

Great catch! Eagle eyes! :bow: